Demo/WhisperDemo/WhisperDemo/ViewController.swift

```diff
@@ -58,7 +58,7 @@ class ViewController: UIViewController {
     return button
     }()
 
-  lazy var containerView: UIView = {
+  lazy var containerView: UIView = { [unowned self] in
```

Comment:
Same here, `self` is not referenced in the closure so it holds no reference to it.